sacred/observers/sql.py

```diff
@@ -20,6 +16,8 @@
 class SqlObserver(RunObserver):
     @classmethod
     def create(cls, url, echo=False, priority=DEFAULT_SQL_PRIORITY):
+        from sqlalchemy.orm import sessionmaker, scoped_session
+        from .sql_bases import sa
```

Comment:
Is there any reason you are import `sqlalchemy` here from `sql_bases` instead of directly?

Reply:
Yes, there is a very good reason. It's because I wasn't attentive, haha.

Reply:
Haha ok. PR looks good, I will merge once the tests pass after the change.